submissions/EmmanuelUdoh/HTMLTASK/login.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Kimaj: for Electronics, Phones, Computers, Fashion </title>
+        <link href="C:\Users\Emmanuel pc\Desktop\Dufuna-CodeCamp19\submissions\EmmanuelUdoh\HTMLTASK\style.css" rel="stylesheet" type="text/css">
```

Comment:
Use the relative path to the CSS file. Just style.css should be fine since the HTML and CSS files are in the same folder.

Why? Because If I clone this code, the CSS files will not be loaded on my computer because I donth have the path "C:\Users\Emmanuel pc\Desktop\Dufuna-CodeCamp19\..."